.github/workflows/rspec_tests.yml

```diff
@@ -0,0 +1,82 @@
+name: Rspec tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Do we need this trigger to be different so it runs on PRs too?

Reply:
Whoops; yeah, I changed it to be `pull_request`.

Reply:
I assume we should also run this on PRs to 2.6.x?

Reply:
Yeah, but I think that should be a different PR to 2.6.x with a different matrix than we will test in master. Puppet seems to keep them separate, I think to ensure that modifying one doesn't modify them all.